modules/users/server/controllers/users.authentication.server.controller.js

```diff
@@ -650,14 +651,24 @@ exports.extendFBAccessToken = function (shortAccessToken, callback) {
  * Confirm email GET from email token
  */
 exports.validateEmailToken = function (req, res) {
+  console.log('🛑 validateEmailToken:'); //eslint-disable-line
   User.findOne(
     {
       emailToken: req.params.token,
     },
     function (err, user) {
       if (!user) {
+        // Add failure tracking
         return res.redirect('/confirm-email-invalid');
       }
+      // Success tracking here
+
+      const isSignup = !user.public;
+      const timeFromEmail = moment().diff(user.emailTokenDate);
```

Comment:
Could be interesting to check if `user.emailTokenDate` is Date object, or does it need to be converted into one with `new Date(user.emailTokenDate)`

Reply:
I'm pretty sure it's working now with moment().diff(moment(user.emailTokenDate))